packages/react/src/components/FeatureFlags/overview.mdx

```diff
@@ -112,6 +114,30 @@ Feature flags can also be enabled via the provided `enable()` mixin
 
 @include feature-flags.enable('enable-experimental-tile-contrast');
 ```
+## Using Codemods for Migration
+
+Codemods are code modification scripts that automate the necessary changes when migrating to new components and APIs. They reduce the manual effort required when adopting new features. Carbon provides codemods to help you enable feature flags and migrate to the new APIs these flags provide.
+
+Some feature flags don't have codemods because:
+- **Style only**: Flags like `enable-experimental-tile-contrast` only affect Sass styles with no changes to the javascript component API, so no code transformation is needed.
+- **Config only**: Flags like `enable-experimental-focus-wrap-without-sentinels`, `enable-dialog-element`, and `enable-treeview-controllable` are configuration options that are enabled in just a few specific places in your application.
+- **App level**: Flags like `enable-v12-dynamic-floating-styles` are designed to be applied at the application level to enable specific behaviors (like floating styles without full autoAlign behavior) and don't require widespread component changes.
```

Comment:
Like we talked about, I think we could merge these? I don't know what title to call it, but the description could be something like

> These flags don't have codemods because their usage can vary widely. Some will opt to turn them on for a single component, some will turn them on for a sub-section(s) of their app/react tree, and some will enable it globally at their root.